app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee

```diff
@@ -86,7 +86,11 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
   editDocument: (e) ->
     # Fire a global event. Consider refactoring this in the style of notifyExpandListeners above.
     Backbone.trigger 'LoanQuestionnairesView:edit', @
-
+    $('.view-element.answer')
+      .next('textarea')
```

Comment:
It makes me a bit nervous to rely on `next`. I'd rather use a single selector by finding its parent or adding a class to the textarea.